File: sdk/python/examples/02_sql_crud.py
"""02_sql_crud:SQL 建表 / 插入 / 查询。"""
from helpers import client

combee = client()
cell = combee.cells.create()
cell.sql.execute("CREATE TABLE users (id INTEGER PRIMARY KEY, name TEXT NOT NULL)")
cell.sql.execute("INSERT INTO users (name) VALUES (?)", ["Alice"])
rows = cell.sql.query("SELECT id, name FROM users")["rows"]
print("users:", rows)

File: sdk/typescript/examples/02_sql_crud.mjs
// 02_sql_crud:SQL 建表 / 插入 / 查询。
import { client } from "./00_helpers.mjs";
const combee = client();
const cell = await combee.cells.create();
await cell.sql.execute("CREATE TABLE users (id INTEGER PRIMARY KEY, name TEXT NOT NULL)");
await cell.sql.execute("INSERT INTO users (name) VALUES (?)", ["Alice"]);
const { rows } = await cell.sql.query("SELECT id, name FROM users");
console.log("users:", rows);
